### DISCUSSION TITLE
Sonoff S20 not resettable after upgade to 6.7.1 minimal

### DISCUSSION THREAD

**Discussion by vguna** [arendst/Tasmota#14195] [Q&A] [answered]
Hey guys,

today I wanted to give the latest Tasmota firmware a try on my Sonoff S20 - which already had 5.9.1 version installed and worked so far (but not used for about 2 years). So as far as I understood, one shouldn't pick the latest for flashing, but do the major upgrades one by one until latest.
So I picked 6.7.1 and tried the flashing via UI by selecting the file. It complained about not enough free space and I picked the minimal version instead. That flashed ok and the S20 restarted. The issue is now, that I can't find it in my Wifi network, as well as there is not dedicated AP available I could connect to. Per default after power on the button LED is green - not blinking. I can turn a connected device on by pressing the button (gets green/blue), BUT it automatically turns it off again after ~42-44 seconds. Then falling back to green steady light.

I already tried several button presses to get it into AP mode again:
- Holding the button longer than 40 seconds
- Pressing the button 4 times quickly
- Pressing the button on/off 7 times... while not turning off the 7th time
- Everything above in different combinations :D.

Now I'm running out of ideas to get it back to life. Do you guys have an idea what might be the issue here?

Thanks!

**Comment by Jason2866** [comment, marked as answer]
Only way to recover is a serial flash.

**Comment by barbudor** [comment]
I don't know what you may done wrong but S20 is easy to open and serial flash
Don't lose time

**Comment by vguna** [comment]
Yeah, finally went that road now. Thanks!
